services/minespace-web/runner/server.js

```diff
@@ -1,7 +1,12 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
 const express = require("express");
+// eslint-disable-next-line @typescript-eslint/no-var-requires
 const cacheControl = require("express-cache-controller");
-const dotenv = require("dotenv").config('./env');
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const dotenv = require("dotenv").config("./env");
+// eslint-disable-next-line @typescript-eslint/no-var-requires
 const expressStaticGzip = require("express-static-gzip");
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
Wait, are you actually getting TS errors in a JS file? That's not normal. 
What was the reason for adding this file to the tsconfig.include? I'm wondering if that's what did it. 🤷‍♀️